index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <head>
+        <link rel="stylesheet" href="style.css">
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Thinus Capstone</title>
+    </head>
+    <body class="BGColor">
+        <!-- col-1 will be used instead of col-xs-1 since bootstrap 5 no longer supports xs screen size -->
+        <nav class="navbar-fluid">
+            <div class="MobileMenu hidden">
+                <ul class="MobileNavBottom row col-12 col-sm-12">
+                    <li class="MobilePopup col-3 col-sm-3 offset-1 offset-sm-1"><a href="pages/about.html">About</a></li>
+                    <li class="MobilePopup col-3 col-sm-3 offset-1 offset-sm-1"><a href="index.html">Program</a></li>
+                    <li class="MobilePopup col-3 col-sm-3 offset-1 offset-sm-1"><a href="#">Join</a></li>
+                    <li class="MobilePopup col-3 col-sm-3 offset-1 offset-sm-1"><a href="#">Sponsor</a></li>
+                    <li class="MobilePopup col-3 col-sm-3 offset-1 offset-sm-1"><a href="#">News</a></li>
+                    <li class="MobilePopup col-3 col-sm-3 offset-1 offset-sm-1"><a href="#">Warframe Lore</a></li>
+                </ul>
+            </div>
+            <div class="NavTop container-fluid">
+                <ul class="row col-lg-4 col-4 col-sm-4 offset-sm-8">
+                    <li class="Socials col-lg-1 col-1 col-sm-1"><a href="index.html"><img class="socials" src="images/fbicon.png" alt="Facebook Icon"></a></li>
+                    <li class="Socials col-lg-1 col-1 col-sm-1"><a href="index.html"><img class="socials" src="images/twittericon.png" alt="Twitter Icon"></a></li>
+                    <li class="NavText col-lg-2 col-2 col-sm-2">English</li>
+                    <li class="NavText col-lg-2 col-2 col-sm-2">My Page</li>
+                    <li class="NavText col-lg-2 col-2 col-sm-2">Logout</li>
+                </ul>
+            </div>
```

Comment:
> * [x]  Kindly ensure that you stick as much as possible to the original design. The background color that you have used for this top bar does not match the one that is provided in the Behance design. 👍 
 * [x]  Great implementing the background, however 👏 
 * [ ]  the social media icons does not intuitively match the design provided in the Behance design. Kindly also make this change. Just the previous review suggested. 🥂  
> 
 **This is how your design currently looks like on the social icons:**
> 
 ![image](https://user-images.githubusercontent.com/86318284/205090478-ccb9d5e2-d457-4dec-8b1d-655eb06372a4.png)
> 
 **This is the desired design:**
> 
 ![image](https://user-images.githubusercontent.com/86318284/205090584-c32ac462-7543-4683-98dc-686db68c5e2d.png)